.github/workflows/edr-npm-release.yml

```diff
@@ -360,6 +360,7 @@ jobs:
   check_commit:
     name: Check commit
     runs-on: ubuntu-latest
+    if: github.event_name != 'pull_request' || github.event.pull_request.author_association == 'OWNER' || github.event.pull_request.author_association == 'MEMBER' || github.event.pull_request.author_association == 'COLLABORATOR'
```

Comment:
@agostbiro this is the best I could do. I don't love it, but I tested it a lot on a test repository and it seems to work fine.

Reply:
Looks good cheers!